requirements/collections_requirements.yml

```diff
@@ -8,5 +8,6 @@ collections:
   - name: openstack.cloud
   - name: community.vmware
   - name: ovirt.ovirt
+  - name: redhat.insights
```

Comment:
Herein lies the issue - the build process cannot handle this automatically right now.

Reply:
Could we publish a _different_ execution environment for this source?

In the bigger picture, I'd like to move away from this single default EE which is not a combination of very unrelated content.

Ideally, we should discover what collections are in an execution environment, and when creating an inventory source, give a listing of execution environments which could plausibly run it. The FQCN `redhat.insights.insights`, which we put into the code itself, gives the collection right there.

Reply:
Eh, I think if we start down the road of one-EE-per-type, and inspecting them, it's a much larger and fundamental change.

Reply:
Setting aside how distant-future it is, or even how much effort it takes, I think one-EE-per-type would be the wrong answer. Some collections, after all, are dependencies of other collections and it wouldn't make sense to cut an image for both. So my idea is 1 EE to several inventory sources (starting whenever a concrete need arises). How many, and how to organize them, I don't know.